docs/meshcloud.project-resources.md

```diff
@@ -3,21 +3,25 @@ id: meshcloud.project-resources
 title: Project Resources
 ---
 
-When a user is [assigned](meshcloud.project.md#assign-user-to-a-meshproject) to a meshProject, this project appears in his project list on the home screen. As long as he is not assigned, he also doesn't have access to the project and its resources. To access the meshProject you can either click on its name or the **Open Project Dashboard** button to get to the [Project Dashboard](#project-dashboard) or you click on a specific Cloud Platform to get directly to the [Project Platform Dashboard](#project-platform-dashboard).
+When users are [assigned](meshcloud.project.md#assign-user-to-a-meshproject) to a meshProject, this project appears in their projects overview on the customer control plane, also known as their home screen. As long as they are not assigned to the project, they do not have access to its resources and the regarding control planes. To access the meshProject you can click on its name to get to the [Project Control Plane](#project-control-plane) or you click on a specific Cloud Platform to get directly to the [Tenant Control Plane](#tenant-control-plane).
```

Comment:
wondering if we should always write Customer Control Plane in upper case as it is a name

Reply:
Exactly. Its not a proper name. So we can write for example customer control plane. Because this is the area to "control/manage" the customer.